bellows/uart.py

```diff
@@ -105,13 +107,15 @@ def rst_frame_received(self, data):
 
     def rstack_frame_received(self, data):
         """Reset acknowledgement frame receive handler"""
-        LOGGER.debug("RSTACK frame: %s", binascii.hexlify(data))
+        LOGGER.debug("RSTACK Version: %d Reason: %s frame: %s", data[1], t.NcpResetCode(data[2]).name, binascii.hexlify(data))
```

Comment:
Since you're creating an `NcpResetCode` twice, I think it might be better to put that in a variable.